### PR TITLE
Update links and CONTRIBUTING

### PR DESCRIPTION
**Pull request by hagenw**
Updates links and CONTRIBUTING to the new links of the package and documentation on Github.

### CODE REVIEW

**Comment by frankenjoe** [comment]
In the documentation we sometimes set the `source` of a `Database` to `https://gitlab.audeering.com/...'. I think we should change that.

File: docs/conf.py [hunk 1]
```diff
@@ -71,10 +68,6 @@
     'python': ('https://docs.python.org/3/', None),
 }
 
-# Disable Gitlab as we need to sign in
-linkcheck_ignore = [
-    'https://gitlab.audeering.com',
-]
 
 # HTML --------------------------------------------------------------------
```
Comment (frankenjoe) [inline docs/conf.py:72]: `'footer_links': False,` is missing in `html_theme_options`

Comment (frankenjoe) [reply docs/conf.py:72]: and we should set:

```
html_context = {
    'display_github': True,
}
```

**Comment by hagenw** [comment]
I updated one Gitlab link in the docs.

We have another one:

```python
    # Create Database
    db = audformat.Database(
        name='databasename',
        source='https://gitlab.audeering.com/data/databasename',
        usage=audformat.define.Usage.COMMERCIAL,
    )
    db
```
But I think there it would make sense to stick with it, as it is still the link that we should use internally for databases.

File: docs/conf.py (hunk 1 above)
Comment (hagenw) [reply docs/conf.py:72]: Good point, added it.

Comment (hagenw) [reply docs/conf.py:72]: OK, added `html_context` as well

**Pull request merged**